### DISCUSSION TITLE
How does one exclude tables from SQL queries.

### DISCUSSION THREAD

**Discussion by mike-niemand** [vanna-ai/vanna#686] [Q&A]
I have some tables in my database e.g. users which I always want to exclude from any generated SQL. I have added this to my training documentation but it still includes it in results.

**Comment by zainhoda** [comment]
Do you have any references to these tables in the training data?

**Reply by mike-niemand** [reply]
Hey Zain. I don't. So should be safe?

**Reply by mike-niemand** [reply]
Only in the schema.
